index.html

```diff
@@ -503,11 +503,11 @@ <h2>The <dfn>ConsumedThing</dfn> interface</h2>
         Promise&lt;void&gt; writeProperty(DOMString propertyName, any value);
         Promise&lt;void&gt; writeMultipleProperties(object valueMap);
         Promise&lt;any&gt; invokeAction(DOMString actionName, optional any params);
-        Promise&lt;void&gt; observeProperty(DOMString name, WotListener listener);
-        Promise&lt;void&gt; unobserveProperty(DOMString name);
-        Promise&lt;void&gt; subscribeEvent(DOMString name, WotListener listener);
-        Promise&lt;void&gt; unsubscribeEvent(DOMString name);
-        readonly attribute ThingDescription td;
+        Promise&lt;DOMString&gt; observeProperty(DOMString name, WotListener listener);
+        Promise&lt;void&gt; unobserveProperty(DOMString subscriptionId);
+        Promise&lt;DOMString&gt; subscribeEvent(DOMString name, WotListener listener);
+        Promise&lt;void&gt; unsubscribeEvent(DOMString subscriptionId);
+        ThingDescription td();
```

Comment:
tend to say `getTD()`

Reply:
Commit amended to use getThingDescription() instead of td()